packages/appeals-service-api/src/lib/notify.js

```diff
@@ -0,0 +1,69 @@
+const { NotifyClient } = require('notifications-node-client');
+const fetch = require('node-fetch');
+const config = require('./config');
+const logger = require('./logger');
+const { getLpa } = require('../services/lpa.service');
+
+function getAddress(siteAddress) {
+  let address = '';
+  if (siteAddress.addressLine1) {
+    address = `${siteAddress.addressLine1}\n`;
+  }
+  if (siteAddress.addressLine2) {
+    address = `${address}${siteAddress.addressLine2}\n`;
+  }
+  if (siteAddress.town) {
+    address = `${address}${siteAddress.town}\n`;
+  }
+  if (siteAddress.county) {
+    address = `${address}${siteAddress.county}\n`;
+  }
+  if (siteAddress.postcode) {
+    address = `${address}${siteAddress.postcode}`;
+  }
+  return address;
+}
+
+async function sendEmail(appeal) {
+  logger.info(`Sending email...`);
```

Comment:
Remove the `...` - can make grepping more awkward

Reply:
OK